doc/error_management.md

```diff
@@ -106,11 +106,11 @@ If you need more information on the errors, or want to act on them in the callin
 
 Sometimes, you want to provide an error code at a specific point in the parser tree. The `add_return_error!` macro can be used for this:
 
-```ignore
-# #[macro_use] extern crate nom;
-# use nom::ErrorKind;
-# use nom::Err::*;
-# use nom::IResult::Error;
+```rust
+#[macro_use] extern crate nom;
```

Comment:
those lines prefixed with `#` are there to hide them in rustdoc's output, so for files in `doc/`, we can just remove them directly